weldx/constants.py

```diff
@@ -1,11 +1,11 @@
 """Define constants for global library use."""
-from pathlib import Path
```

Comment:
I think using `Path` is common here, any collisions?

Reply:
It did not cause any collisions, I just wanted to cleanup the namespace a bit. Some IDEs/frontends also show all "public" attributes of a module and ignore the meaning of `__all__`